src/components/SidebarButton.js

```diff
@@ -2,19 +2,19 @@ import React from 'react';
 import classNames from 'classnames';
 import camera from './icons/camera.svg';
 import preview from './icons/preview.svg';
-import reset from './icons/reset.svg';
 import download from './icons/download.svg';
 import burst from './icons/burst.svg';
 import settings from './icons/settings.svg';
+import trash from './icons/trash.svg';
```

Comment:
Though the icon is a trash can, not sure if Chris wants to maintain the import of 'reset' to indicate what the action is _doing_ later in the code, rather than what the svg itself is.

Reply:
Good call. I just changed it to 'reset'.